install.sh

```diff
@@ -44,15 +44,11 @@ ${RESET}"
 
 echo ""
 echo "Installing docker images for ddev to use..."
-if [ ${OS} = "MINGW64_NT-10.0" ] ; then PATH="./bin/windows:$PATH"; fi
 
-
-if command -v 7z >/dev/null; then
-    7z x ddev_tarballs/ddev_docker_images.${DDEV_VERSION}.tar.xz -so | docker load
-elif command -v xzcat >/dev/null; then
+if command -v xzcat >/dev/null; then
```

Comment:
I'm not sure this is right. I didn't see xzcat bundled with Git-Bash (gzcat, bzcat, zcat are). I only found that `tar` supported xz.

Reply:
I'll have to check tomorrow as I don't have access to a Windows machine.

Reply:
Yes, xzcat is in git 2.18.0, yay!

Reply:
Looks like from the release notes they added it (and tar capability) in 2.16.2.

Reply:
Oh, I found it. Most of the other utilities are in `/usr/bin` but xzcat is in `/mingw64/bin`.